Aptoide/aptoide-client-v8
# [MOB-75] Migrate experiment to room

**What does this PR do?**

   This PR aims at implementing room to cache the ab test experiments.

**Database changed?**

   No

**Where should the reviewer start?**

- [x] ABTestCenterRepository.java

**How should this be manually tested?**

Open the app and by looking at charles requests you can see that it is being cached. The first time it will make a request to get the info, after that it won't.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-75](https://aptoide.atlassian.net/browse/MOB-75)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
